lib/src/robot/sessions_client.dart

```diff
@@ -55,8 +55,8 @@ class SessionsClient implements ResourceRPCClient {
         // We send heartbeats slightly faster than the interval window to
         // ensure that we don't fall outside of it and expire the session.
         _heartbeatInterval = Duration(
-          seconds: response.heartbeatWindow.seconds.toInt() ~/ 5,
-          microseconds: response.heartbeatWindow.nanos ~/ 5,
+          seconds: response.heartbeatWindow.seconds.toInt() ~/ 1.8,
```

Comment:
How did you land on 1.8?

Reply:
good question, i was playing with 2, and wanted slightly slower than 2. I don't think 1.8 makes any large difference from 2 and probably gets lost most of the time staying as an int. Basically, there is not hard and fast reason why i landed on 1.8